bundler/lib/bundler/man/bundle-inject.1.ronn

```diff
@@ -19,4 +19,6 @@ Example:
     bundle inject 'rack' '> 0'
 
 This will inject the 'rack' gem with a version greater than 0 in your
-[`Gemfile(5)`][Gemfile(5)] and Gemfile.lock
+[`Gemfile(5)`][Gemfile(5)] and Gemfile.lock.
+
+The `bundle inject` command was deprecated in Bundler 2.1 and will be removed in Bundler 3.0.
```

Comment:
Any specific reason to not mention (similar to the deprecation message) it was replaced with `bundle add`?

Reply:
I could not recall how to write the recommended ways for the deprecated commands in `man`s. While I have checked it out at https://manpages.debian.org/unstable/apt/apt-key.8.en.html, should we add the DEPRECATION section to bundle-inject(1), for example?

Unlike the suggestion from the deprecation message in the CLI, I could not see `bundle add` is the in-replacement of `bundler inject` since `bundle inject thor 1.0` can be replaced with `bundle add thor 1.0` (we should do `bundle add thor --version 1.0`).

Reply:
Ahh sorry, it is part of https://github.com/rubygems/rubygems/blob/96e5cff3df491c4d943186804c6d03b57fcb459b/bundler/UPGRADING.md actually.

Reply:
So good to merge?

Reply:
@deivid-rodriguez I am good to merge from my side.

Reply:
Alright, we can always add further improvements later! Thanks!